src/Core/src/Core/IGraphicsView.cs

```diff
@@ -17,12 +17,46 @@ public interface IGraphicsView : IView
 		/// </summary>
 		void Invalidate();
 
+		/// <summary>
+		/// Raised when a pointer enters the hit test area of the GraphicsView.
+		/// </summary>
+		/// <param name="points">The set of positions where there has been interaction.</param>
 		void StartHoverInteraction(PointF[] points);
+
+		/// <summary>
+		/// Raised when a pointer moves while the pointer remains within the hit test 
+		/// area of the GraphicsView.
+		/// </summary>
+		/// <param name="points">The set of positions where there has been interaction.</param>
 		void MoveHoverInteraction(PointF[] points);
+
+		/// <summary>
+		/// Which is raised when a pointer leaves the hit test area of the GraphicsView.
```

Comment:
We should be consistent in the style here, so either start with "Raised when" every time, or "Which is raised" every time. (I prefer "Raised when".)